### PR TITLE
New package: NetEase.Minecraft version 1.17.5.8

### PR DESCRIPTION
**Pull request by DuckDuckStudio**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/199536)

### CODE REVIEW

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 12:04:24 PM: 
> 2024-12-18 12:03:30.172   Failed to proceed to next wizard page; aborting.

(Automated response - build 931.)

**Comment by DuckDuckStudio** [comment]
> Edit: It appears that no executables were added to the file system. Based on the error, I'm guessing there was an installer error. Is something like this expected?

No.

File: manifests/n/NetEase/Minecraft/1.17.5.8/NetEase.Minecraft.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: '{17EF2B1F-E35E-414E-850A-D45E7D4717F4}_is1'
+ReleaseDate: 2024-12-17
+#AppsAndFeaturesEntries: --> By komac
+#- DisplayName: 网易发烧游戏
+#  Publisher: Netease
+#  ProductCode: '{17EF2B1F-E35E-414E-850A-D45E7D4717F4}_is1'
```
Comment (DuckDuckStudio) [inline manifests/n/NetEase/Minecraft/1.17.5.8/NetEase.Minecraft.installer.yaml:18]:
```suggestion
AppsAndFeaturesEntries:
- DisplayName: 网易发烧游戏
  Publisher: Netease
  ProductCode: '{17EF2B1F-E35E-414E-850A-D45E7D4717F4}_is1'
```
Maybe komac is correct...

**Comment by DuckDuckStudio** [comment]
It looks like this package doesn't support silent installation, or more parameters need to be added for silent installation, but I wasn't able to find out exactly what more parameters to add.
What is certain is that this is an inno installer.

Hint for people who want to add this package later: this is most likely due to not agreeing to this package's agreement in the parameters.


<details>
<summary>原文</summary>
看起来此软件包不支持静默安装，或者需要添加更多参数才可以静默安装，但我没能找到具体要再添加什么参数。<br>
可以确定的是，这是一个inno安装程序。<br>

对于以后想要添加此软件包的人的提示：这很可能是因为没有在参数中同意协议。
</details>

File: manifests/n/NetEase/Minecraft/1.17.5.8/NetEase.Minecraft.locale.zh-CN.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Tags:
+- 3D
+- Minecraft
+- 开放世界游戏
+- 我的世界
+- 方块
+- 沙盒游戏
+- 游戏
+- 网易
+- 网易游戏
```
Comment (DuckDuckStudio) [inline manifests/n/NetEase/Minecraft/1.17.5.8/NetEase.Minecraft.locale.zh-CN.yaml:27]:
```suggestion
- 网易游戏
Agreements:
  - AgreementLabel: 网易发烧游戏-用户协议与隐私政策
    AgreementUrl: https://fever.163.com/rule/#/
  - AgreementLabel: 《我的世界》正式版用户协议
    AgreementUrl: https://protocol.unisdk.netease.com/release/latest_v15.html
```

**Pull request closed without merging**